packages/reveal-latte/src/Rules/LatteCompleteCheckRule.php

```diff
@@ -169,9 +174,8 @@ private function processTemplateFilePath(
                 $scope,
                 $componentNamesAndTypes
             );
-        } catch (Throwable) {
-            // missing include/layout template or something else went wrong → we cannot analyse template here
-            $errorMessage = sprintf('Template file "%s" does not exist', $templateFilePath);
```

Comment:
this was my error message but I realize that real message from catched Error / Exceptiion would be much better